packages/e2e/.env.example

```diff
@@ -1,11 +1,19 @@
 # Logger
 LOGGER_MIN_SEVERITY=info
 
-# Providers setup
+# Faucet configuration
 FAUCET_PROVIDER=cardano-wallet
 FAUCET_PROVIDER_PARAMS='{"baseUrl":"http://localhost:8090/v2","mnemonic":"fire method repair aware foot tray accuse brother popular olive find account sick rocket next"}'
+
+# Key management setup - required by getWallet
 KEY_MANAGEMENT_PROVIDER=inMemory
 KEY_MANAGEMENT_PARAMS='{"accountIndex": 0, "networkId": 0, "password":"some_password","mnemonic":"vacant violin soft weird deliver render brief always monitor general maid smart jelly core drastic erode echo there clump dizzy card filter option defense"}'
+#KEY_MANAGEMENT_PARAMS='{"accountIndex": 0, "networkId": 0, "password":"some_password","mnemonic":"slab gorilla reflect display cage aim silver add own arrange crew start female bitter menu inner combine exit swallow bamboo midnight wealth culture picnic"}'
+#KEY_MANAGEMENT_PARAMS='{"accountIndex": 0, "networkId": 0, "password":"some_password","mnemonic":"decorate survey empower stairs pledge humble social leisure baby wrap grief exact monster rug dash kiss perfect select science light frame play swallow day"}'
```

Comment:
Is the duplication of `KEY_MANAGEMENT_PARAMS` intentional?

Reply:
Oh, it's documenting the available mnemonics?

Reply:
Yes @rhyslbw , it is.
They are the the 5 mnemonics of the 5 funded accounts. The purpose is to make easier to change between a funded account and another one.